source/requirements.txt

```diff
@@ -48,10 +48,10 @@ sphinxext-delta==0.1.0; python_version >= "3.6"
 sphinxext-linkcheckdiff==0.1.0; python_version >= "3.6"
 sphinxext-opengraph==0.4.1; python_version >= "3.6"
 sphinxext-rediraffe==0.2.5; python_version >= "3.6"
-sphinxext-remoteliteralinclude==0.2.0; python_version >= "3.4"
+sphinxext-remoteliteralinclude==0.3.0; python_version >= "3.4"
 sphinxext-toptranslators==0.1.2; python_version >= "3.6"
 stevedore==3.3.0; python_version >= "3.6"
 tempdir==0.7.1
 typing-extensions==3.7.4.3; python_version < "3.8" and python_version >= "3.6"
-urllib3==1.26.4; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.5.0" and python_version < "4" and python_version >= "3.6"
+urllib3==1.26.4; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.5.0" and python_version < "4")
```

Comment:
Doesn't this still cause us to use an old urllib3?

Reply:
Huh, interesting. Fixed.